test/ex_doc/formatter/html/templates_test.exs

```diff
@@ -463,4 +463,33 @@ defmodule ExDoc.Formatter.HTML.TemplatesTest do
                ~r{<h1>\s*mix task_with_docs\s*<small class=\"app-vsn\">\(Elixir v1.0.1\)</small>}
     end
   end
+
+  describe "bottom actions" do
```

Comment:
I was hoping more of an integration test, perhaps in html_test, as this one does not really assert we have built the refs properly (which is the complex part!). Do we have two guides in our test suite? It should be enough that generating docs for our existing guides and match on the output.

Reply:
Sure, should've done that before!